music/signergroupops.go

```
@@ -194,185 +202,192 @@ COALESCE(pendremove, '') AS prem, locked FROM signergroups`
-				}
-				if signer == "" { // There may be rows with signer=="" (if group created w/o signers)
-					continue
-				}
-				s, err := mdb.GetSignerByName(tx, signer, true) // apisafe
-				if err != nil {
-					log.Fatalf("ListSignerGroups: Error from GetSigner: %v", err)
-				} else {
-					signers[signer] = s
-				}
+		var signer string
+		var zones = []*Zone{}
```

golangci-lint (1.62.2)

ineffectual assignment to zones

(ineffassign)